File: Server/HiSql.GUI/Repository/Models/HiExcelHeaderMapEntity.cs
using System;
namespace HiSql.GUI.Repository.Models
{
    public class HiExcelHeaderMapEntity
    {

        /// <summary>
        /// MapId[必填参数]
        /// <summary>
        public string MapId { get; set; }

        /// <summary>
        /// TemplateId
        /// <summary>
        public string TemplateId { get; set; }

        /// <summary>
        /// SheetName
        /// <summary>
        public string SheetName { get; set; }

        /// <summary>
        /// HeadName
        /// <summary>
        public string HeadName { get; set; }

        /// <summary>
        /// FieldName
        /// <summary>
        public string FieldName { get; set; }

    }
}

File: Web/src/serverApi/models/excelModel/HiExcelHeaderMapEntity.ts
import { StringHelper } from '@/helper/stringHelper';

export class HiExcelHeaderMapEntity {
  /**
   * MapId[必填参数]
   */
  MapId: string = StringHelper.genGuid();

  /**
   * TemplateId
   */
  TemplateId = ``;

  /**
   * SheetName
   */
  SheetName = '';

  /**
   * HeadName
   */
  HeadName = '';

  /**
   * FieldName
   */
  FieldName = '';

  /**
   * 列索引
   */
  SortNum = 0;
}
